extensions/amp-story-captions/0.1/amp-story-captions-presets.css

```diff
@@ -0,0 +1,46 @@
+/* default */
```

Comment:
Please import the shadow-reset css, otherwise inherited styles such as `letter-spacing` will still change the captions display, even within a shadow root.